content/sensu-go/5.17/release-notes.md

```diff
@@ -51,6 +52,26 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.17.1 release notes
+
+**January 31, 2020** &mdash; The latest release of Sensu Go, version 5.17.1, is now available for download.
+This release fixes a web UI issue that cleared selected filters when sorting an event list and a bug that prevented certain `.tar` assets from extracting.
+It also includes sensuctl configuration improvements.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.17.1.
+
+**IMPROVEMENTS:**
+
+- Asset names may now include capital letters.
+- Running the `sensuctl configure` command now resets the sensuctl cluster configuration.
+- When you use `--trusted-ca-file` to configure sensuctl, it now detects and saves the absolute file path in the cluster configuration.
+
+**FIXES:**
+
+- ([Commercial feature][106]) In the [web UI][107], polling is re-enabled for the silences view.
```

Comment:
I believe this change should be described differently. As I understand it from https://github.com/sensu/sensu-enterprise-go/issues/853, users may have noticed silencing entries lingering in the web UI after they had been removed. The fix ensures that the web UI is updated when a silencing entry is removed or expires.